libafl/src/executors/inprocess/mod.rs

```diff
@@ -454,6 +454,16 @@ pub fn run_observers_and_save_state<E, EM, OF, Z>(
         let mut new_testcase = Testcase::with_executions(input.clone(), executions);
         new_testcase.add_metadata(exitkind);
         new_testcase.set_parent_id_optional(*state.corpus().current());
+
+        match state.current_testcase_mut() {
+            Ok(mut tc) => {
+                tc.set_objectives_found(true);
+            }
+            Err(_) => {
+                // just do nothing, we are about to die. returning error makes no sense
```

Comment:
Always log the errors

Reply:
ok.
but i have the fundamental question. is it safe to access current_testcase_mut() here?

Reply:
Should be ok _I think_